src/app/core/ui/sync-status/background-processing-indicator/background-processing-indicator.component.scss

```
@@ -11,3 +11,12 @@
 .details-line {
   line-height: 32px;
 }
+
+.multiline {
+  white-space: normal;
+  overflow: visible;
+  word-break: break-word;
```

Stylelint (17.11.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)